ChainofResponsability/src/Demo.java

```diff
@@ -0,0 +1,27 @@
+import java.util.HashMap;
+
+public class Demo {
+    public static void main(String[] args) {
+        HashMap<String, String > inputs = new HashMap<>();
+        inputs.put("email", "malau@gmail.com");
+        inputs.put("integer", "7343650");
+        inputs.put("date", "01/01/2011");
+
+        Inputs dataReceived = new Inputs(inputs);
+
+        ISanitization emailSanitizer = new EmailSanitizer(inputs.get("email"));
```

Comment:
this is odd, why do you need to pass values here ? the whole intention is to send inputs to the sanitizer to provide a sanitized version of the output with the appropriate types, so for sanitizer, you receive `inputs ` and you got in return `output` and output should have the appropriate values with types, email as string, integer as int and date as Date